11_taxonomic_information.Rmd

```diff
@@ -185,12 +278,15 @@ such as the centered log-ratio transformation (clr).
 
 In mia package, transformations are applied to abundance data. The transformed 
 abundance table is stored back to 'assays'. mia includes transformation 
-function ('transformAssay()') which applies sample-wise or column-wise transformation when MARGIN = 'samples', feature-wise or row-wise transformation when MARGIN = 'features'.
+function ('transformAssay()') which applies sample-wise or column-wise 
+transformation when MARGIN = 'samples', feature-wise or row-wise transformation 
+when MARGIN = 'features'.
 
 For a complete list of available transformations and parameters, see function 
 [help](https://microbiome.github.io/mia/reference/transformAssay.html).
 
 ```{r}
+tse <- tseGlobalPatterns
 tse <- transformAssay(tse, assay.type = "counts", method = "relabundance", pseudocount = 1)
 tse <- transformAssay(x = tse, assay.type = "relabundance", method = "clr", 
                         pseudocount = 1, name = "clr_transformation")
```

Comment:
"clr" would be enough for a good assay name ("clr_transformation" is a bit long to write when using these)